pyproject.toml

```
@@ -0,0 +1,138 @@
+[build-system]
+requires = ["setuptools>=66.1", "wheel", "Cython>=3", "fastrlock>=0.5"]
```

setuptools v77 required for `project.license-files`.

```suggestion
requires = ["setuptools>=77", "wheel", "Cython>=3", "fastrlock>=0.5"]
```